doc/api/n-api.md

```diff
@@ -4188,6 +4189,11 @@ prevent the event loop from exiting. The APIs `napi_ref_threadsafe_function` and
 
 <!-- YAML
 added: v8.16.0
+napiVersion: 4
+changes:
+  - version: v10.17.0
```

Comment:
I don't quite understand this being version: v10.17.0 for the version 8 backport.  Also makes me wonder about what we have in head.  I'd expect that once this gets landed in head we might want to update master to reflect the version it was landed in the 8.x stream?

Reply:
```suggestion
  - version: REPLACEME
```

This should have been `REPLACEME` on master.